<launch>
    <node name="theodolite_interactive_master" type="theodolite_interactive_master" pkg="theodolite_node" output="screen">

        <!-- Parameter to show data on terminal if needed. Set True to use it -->
        <param name="show_data" value= "true" />

	</node>
</launch>
